docs/contributor-guide/developer-guide/feature-flags.md

```diff
@@ -34,6 +39,15 @@ The following process should be used to implement a feature flag, release it in
 11. Leave the flag in the codebase for at least another month after it went live in production (e.g. Seattle).
 12. Remove the flag, remove the guards in the code looking for the flag, note in the PR release notes that the flag is being removed, then close the Github issue. Ensure the email sent for the next release includes a note about the removal of the feature flag.
 
+
+### Adding a new flag
+
+1.  Add the flag to the `"Feature Flags"` section of [server/conf/env-var-docs.json](https://sourcegraph.com/github.com/civiform/civiform/-/blob/server/conf/env-var-docs.json). 
+1.  Run `bin/fmt` to regenerated `SettingsManifest.java`
+1.  Inject `SettingsManifest` where you want to consume the flag
+1.  Call the generated method on `SettingsManifest` that matches the name of your flag. Unless infeasible, use the version of the method that takes a `play.mvc.Http.Request` argument so that the flag can be overridden in non-prod environments.
```

Comment:
suggestion (non-blocking): give an example of a flag + how to invoke the getter method for it

Reply:
Done